sdk/cosmosdb/cosmos/samples-dev/ItemManagement.ts

```diff
@@ -5,13 +5,12 @@
  * @summary Demonstrates item creation, read, delete and reading all items belonging to a container.
  */
 
-import * as dotenv from "dotenv";
-dotenv.config();
+import "dotenv/config";
 
-import { logSampleHeader, handleError, finish, logStep } from "./Shared/handleError";
-import { CosmosClient, PatchOperation, PriorityLevel } from "@azure/cosmos";
-
-import { Families } from "./Data/Families.json";
+import { logSampleHeader, handleError, finish, logStep } from "./Shared/handleError.js";
+import type { PatchOperation } from "@azure/cosmos";
+import { CosmosClient, PriorityLevel } from "@azure/cosmos";
+import Families from "./Data/Families.json" assert { type: "json" };
```

Comment:
I think we should the stable syntax in https://github.com/Azure/azure-sdk-for-js/pull/32306

Reply:
Fixed